pymobiledevice3/services/dvt/instruments/core_profile_session_tap.py

```diff
@@ -671,7 +677,7 @@ def parse_stackshot(data):
         return parsed_stack_shot[predefined_names[kcdata_types_enum.KCDATA_BUFFER_BEGIN_STACKSHOT]]
 
     @staticmethod
-    def get_time_config(dvt):
+    def get_time_config(dvt: 'DvtSecureSocketProxyService') -> Mapping[str, Union[int, timezone, Any]]:
```

Comment:
Union is meaningless when using `Any`, just use `Any`

Reply:
k